applications/FluidDynamicsApplication/symbolic_generation/compressible_navier_stokes/generate_compressible_navier_stokes_explicit_with_integration.py

```diff
@@ -0,0 +1,303 @@
+from KratosMultiphysics import *
+from KratosMultiphysics.sympy_fe_utilities import *
+
+from sympy import *
+import pprint
+
+from params_dict import params
+import generate_convective_flux
+import generate_diffusive_flux
+import generate_source_term
+import generate_stabilization_matrix
+
+# dim = params["dim"]         # Define Dimension in params.py
+do_simplifications = False
```

Comment:
in a future PR, when it works correctly, we should have a True here

Reply:
I switched it to `True` and after one night trying to generate the element it didn't finished yet. I'd leave it as `False`. In the final steps of the implementation we can switch it on and generate the final version of the cpp.

Reply:
ok